docs/source/python/ipc.rst

```diff
@@ -80,6 +80,7 @@ this one can be created with :func:`~pyarrow.ipc.new_stream`:
    with pa.ipc.new_stream(sink, batch.schema) as writer:
       for i in range(5):
          writer.write_batch(batch)
+      writer.close()
```

Comment:
I think that exiting from the context should call this automatically? (that could also be mentioned in the text to make this explicit)